test/Arduino.h

```diff
@@ -13,12 +14,23 @@ struct Print
         buf << obj;
     }
 
+    template <typename T>
+    void println(const T &obj)
+    {
+        buf << obj << std::endl;
+    }
+
     void begin(int)
     {
         buf << std::fixed << std::showpoint << std::setprecision(2);
         buf.str("");
     }
 
+    Print& operator<< (std::ostream& (*pf)(std::ostream&)){
+        buf << pf;
```

Comment:
```suggestion
    Print& operator<< (std::ostream& (*pf)(std::ostream&))
    {
        buf << pf;
```
Sorry to be a stickler, but could we move that open bracket to a new line? Also just to confirm, I guess the idea with this function is to allow a caller to pass things like `std::endl` to `Print` right?